server/cpho/views/indicators.py

```diff
@@ -95,3 +118,259 @@ def get_context_data(self, **kwargs):
         return {
             **super().get_context_data(**kwargs),
         }
+
+
+class UploadIndicator(FormView):
+    template_name = "indicators/upload_indicator.jinja2"
+    form_class = UploadForm
+    # success_url = "/upload_indicators/"
+
+    def get_success_url(self):
+        return reverse("list_indicators")
+
```

Comment:
you probably won't need it at all, but you can override form_invalid if you want to use something like messages